docs/section5/Rule5-1.md

```diff
@@ -0,0 +1,73 @@
+
+**Evaluation Context:** Each RMD 
+
+**Applicability Checks:**  
+None
+   
+**Function Calls:**  
+1. get_opaque_surface_type()  
+
+
+## Rule Logic:   
+
+Get the fenestration area for each unique orientation (i.e., azimuth) and then check if the minimum and maximum areas differ by 5% or more
+- Create a blank dictionary that will have all unique azimuths paired with the total vertical fenestration area: `azimuth_fen_area_dict = {}`  
+- For each building in the B_RMI: `for bldg in B_RMI.buildings:`    
+    - For each building_segment in the bldg: `for bldg_seg in bldg.building_segments`      
+        - For each zone in the building_segment: `for zone in bldg_seg.zones`   
+            - For each surface in zone: `for surface in zone.surfaces:`  
+                - Check if surface is above-grade wall: `if get_opaque_surface_type(surface) == "ABOVE-GRADE WALL":`   
+                    - Get the azimuth: `surface_azimuth = surface.azimuth`  
+                    - Check if the azimuth is not in a key in the dictionary of azimuths and fen area, if it is not then add it: `if surface_azimuth not in azimuth_fen_area_dict:`   
+                        - Add the unique azimuth to the dictionary as a key and set initial fen area to 0: `azimuth_fen_area_dict[surface_azimuth]= 0`  
+                    - Reset the total surface fenestration area variable to 0: `total_surface_fenestration_area = 0`  
+                    - For each subsurface associated with the surface: `for sub_surface in surface.subsurfaces:`   
+                        - Check if subsurface is door: `if sub_surface.classification == "DOOR":`
+                            - If glazed area in door is more than 50% of the total door area, add door area to total_surface_fenestration_area: `if sub_surface.glazed_area > subsurface.opaque_area: total_surface_fenestration_area += sub_surface.glazed_area + sub_surface.opaque_area`
+                        - Else, subsurface is not door, add total area to total_surface_fenestration_area (because we checked that the surface is an above grade wall we can assume that the subsurface classification is not and could not be a skylight so no need to check this): `total_surface_fenestration_area += subsurface.glazed_area + subsurface.opaque_area`      
+                    - Add the total_surface_fenestration_area summed for the surface to the total fen area associated with the azimuth: `azimuth_fen_area_dict[surface_azimuth] += total_surface_fenestration_area`    
+
+Check if the area differs by 5 percent or more.
+- Get the max fen area: `max_fen_area = azimuth_fen_area_dict[max(azimuth_fen_area_dict, key=azimuth_fen_area_dict.get)]`  
+- Get the min fen area: `min_fen_area = azimuth_fen_area_dict[min(azimuth_fen_area_dict, key=azimuth_fen_area_dict.get)]`  
+- Calculate the % difference, take the maximum calculated: `percent_difference = max(abs(max_fen_area- min_fen_area)/max_fen_area,abs(min_fen_area- max_fen_area)/min_fen_area)` 
```

Comment:
Since 90.1 does not say how to determine the percentage difference I would assume both of the following scenarios to allow not rotating:
a. (max_fen_area-min_fen_area)/max_fen_area<5%
b. (max_fen_area-min_fen_area)/min_fen_area<5%

Reply:
@mkarpman So more lenient then?

Reply:
I think the more lenient approach would be to not report error if EITHER (a) or (b) are true.

Reply:
@mkarpman I think what you suggest then is already implemented since it uses the max of both calculations .

Reply:
@supriyagoel It looks like it includes both in the max function. Can you explain what you are referring to? Thanks!

Reply:
@mkarpman I agree with No. 1 since I think the assumption with the exception is that rotating is not impactful so why should it matter if they rotate when not required. I modified the RDS accordingly. If anyone disagrees please chime in.

Reply:
@claperle this is the line i'm talking about -  Calculate the % difference, take the maximum calculated: `percent_difference = max(abs(max_fen_area- min_fen_area)/max_fen_area,abs(min_fen_area- max_fen_area)/min_fen_area)` 

but I noww see that we're taking the absolute - so please ignore my comment.